prometheus-exporters-formula/prometheus-exporters-formula.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Oct  8 13:23:59 UTC 2020 - Dario Maiocchi <dmaiocchi@suse.com>
+
+- Version 0.7.5:
```

Comment:
Could you please bump the version number to 0.8.0? I'll package that together with https://github.com/SUSE/salt-formulas/pull/82

Reply:
ok